### PR TITLE
tests: fix for tests/main/lxd-mount-units test and enable ubuntu-21.04

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Some lxd images are comming with core20 installed instead of core18

### CODE REVIEW

File: tests/main/lxd-mount-units/task.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ details: |
 
 # only 20.04+, we want lxd images that come with snaps preinstalled.
 # TODO: enable for ubuntu-21+ once the lxd image is published
```
Comment (bboozzoo) [inline tests/main/lxd-mount-units/task.yaml:8]: I guess the TODO can be dropped now, but we should land the PR first.

**Review by bboozzoo — APPROVED** [review]
Thank you!

File: tests/main/lxd-mount-units/task.yaml [hunk 2]
```diff
@@ -46,13 +52,13 @@ execute: |
...
-    lxd.lxc exec ubuntu -- find /var/run/systemd/generator/ -name container.conf | MATCH "/var/run/systemd/generator/snap-core18.*mount.d/container.conf"
+    lxd.lxc exec ubuntu -- find /var/run/systemd/generator/ -name container.conf | MATCH "/var/run/systemd/generator/snap-${core_snap}.*mount.d/container.conf"
     lxd.lxc exec ubuntu -- test -f /var/run/systemd/generator/snap.mount
 
-    echo "Sanity check that core18 snap is mounted correctly"
-    # Make sure core18 is mounted and readable for a regular user
-    retry -n 5 --wait 1 sh -c 'lxd.lxc exec ubuntu -- sudo --user ubuntu --login ls /snap/core18/current'
-    retry -n 5 --wait 1 sh -c 'lxd.lxc exec ubuntu -- sudo --user ubuntu --login mount | grep core18| grep allow_other'
+    echo "Sanity check that $core_snap snap is mounted correctly"
+    # Make sure $core_snap is mounted and readable for a regular user
```
Comment (bboozzoo) [inline tests/main/lxd-mount-units/task.yaml:59]: nitpick, $ does not expand in the comments, but maybe it's clear that this is about whatever is the value of $core_snap

**Review by stolowski — APPROVED** [review]
Thanks!

**Pull request merged**